Apps/Admin/Tests/Functional/cypress.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = defineConfig({
             idir_password: "",
             keycloak_username: "blazoradmin",
             keycloak_password: "",
+            unauthorized_keycloak_username: "healthgateway",
```

Comment:
Can we change this to 'keycloak_unauthorized_username'?  This seems to be the standard for Healthgateway functional tests.

<img width="577" alt="Screen Shot 2022-08-15 at 3 29 33 PM" src="https://user-images.githubusercontent.com/58790456/184731777-331a2fe9-498e-49a9-95a2-81fde6257f79.png">